tests/agent/test_host.py

```diff
@@ -64,7 +64,7 @@ def test_has_options(self) -> None:
         assert isinstance(self.agent.options, StandardOptions)
 
     def test_agent_default_log_level(self) -> None:
-        assert self.agent.options.log_level == logging.WARNING
+        assert self.agent.options.log_level == logging.WARN
```

Comment:
```suggestion
        assert self.agent.options.log_level == logging.WARNING
```
Why was this change made? I can see that they are essentially the same.
```
WARNING = 30
WARN = WARNING
```